itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTest.java

```diff
@@ -156,6 +156,37 @@ public static void refreshProject(String projectName, String projectType) {
 			new ContextMenuItem(TextLabels.GRADLE_CONTEXT_MENU_ITEM, TextLabels.REFRESH_GRADLE_PROJECT).select();
 		}
 		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		if (projectType.equals(TextLabels.MAVEN_TYPE)) {
+			refreshProjectWorkaround(projectName);
+		}
+	}
+	
+	private static void refreshProjectWorkaround(String projectName) {
+		try {
+			Thread.sleep(10000);
```

Comment:
Instead of Thread.sleep use something more specific 
`new WaitUntil(new JobIsRunning, TimePeriod.DEFAULT, false)`
or
`AbstractWait.sleep` is there is nothing specific to wait for.

Reply:
done